sdk/resourcemanager/azure-resourcemanager-samples/src/main/java/com/azure/resourcemanager/appplatform/samples/ManageSpringCloud.java

```diff
@@ -72,13 +72,13 @@
      * @throws IllegalStateException unexcepted state
      */
     public static boolean runSample(AzureResourceManager azureResourceManager, String clientId) throws IOException, KeyStoreException, CertificateException, NoSuchAlgorithmException {
-        final String rgName = azureResourceManager.sdkContext().randomResourceName("rg", 24);
-        final String serviceName  = azureResourceManager.sdkContext().randomResourceName("service", 24);
+        final String rgName = azureResourceManager.resourceGroups().manager().sdkContext().randomResourceName("rg", 24);
```

Comment:
randomResourceName in samples is quite a mess. Maybe not now, but need to find a better way.